src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -221,7 +221,8 @@ private boolean containsJavadocTag(final TextBlock javadoc) {
                     found = true;
                 }
                 else if (noArgMultilineStart.find()) {
-                    found = checkTagAtTheRestOfComment(lines, found, currentLine, i);
+                    checkTagAtTheRestOfComment(lines, found, currentLine, i);
+                    found = true;
```

Comment:
By the time we get here, a javadoc is always found, like the above if portion and I never found an instance where it returned false.

I will recheck regression with `skipNoJavadoc` = true.

Reply:
confirmed. 
I was suspicious at first look, but it is ok.
This Check implementation is not based on ANTLR grammar so, any small regression is ok.